lib/utils/index.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
    *
    * @param {RuleContext} context The rule context to use parser services.
    * @param {Object} templateBodyVisitor The visitor to traverse the template body.
-   * @param {Object} scriptVisitor The visitor to traverse the script.
+   * @param {Object} [scriptVisitor] The visitor to traverse the script.
```

Comment:
Why square brackets here?

Reply:
It indicates an `optional param` http://usejsdoc.org/tags-param.html#optional-parameters-and-default-values